autoload/neomake.vim

```diff
@@ -248,10 +248,12 @@ function! s:Make(options) abort
         endif
     endif
 
+    let title = join(map(copy(enabled_makers), 'v:val.name'), ', ')
```

Comment:
@nerdrew 
Seen this error here:
```
Error detected while processing function neomake#Make[13]..<SNR>194_Make:
line   17:
E121: Undefined variable: name
```
Just by running `Neomake` on `plugin/neomake.vim` on this PR.